errbot/backends/irc.py

```diff
@@ -248,6 +277,9 @@ def topic(self, topic):
         :param topic:
             The topic to set.
         """
+        if not self.joined:
+            raise RoomNotJoinedError("Must join the room to set the topic")
+        self.cb_set_topic(topic)
```

Comment:
yeah, My bad, Mixed up stage and branch and finish with some crazy stuff around. I will fix this and others and update the pull, sorry for make you work on this mixup.